DecipherNow/helm-charts
# use global.zone

Resolves #441 

## Changes
- Replace hardcoded `zone-default-zone` with `{{ .Values.global.zone }}`

## Test Instructions
- In `greymatter.yaml`, change global.zone to something other than `zone-default-zone` and then run `make dev`. Make sure everything starts up and you can hit the dashboard.

NOTE: I did not update the data-standalone deployment as it doesn't use the standard helm templating like the rest of the repo. This would probably be a bigger refactor for a follow up PR. https://github.com/DecipherNow/helm-charts/issues/469
